backend/prompt_studio/prompt_studio_core_v2/prompt_studio_helper.py

```diff
@@ -780,7 +780,7 @@ def dynamic_indexer(
         vector_db = str(profile_manager.vector_store.id)
         x2text_adapter = str(profile_manager.x2text.id)
         extract_file_path: Optional[str] = None
-
+        directory, filename = os.path.split(file_path)
```

Comment:
@johnyrahul Handled this change in this PR - https://github.com/Zipstack/unstract/pull/720